test/_test-data.js

```diff
@@ -1,4 +1,5 @@
-import { PrivateKey, constants } from "../";
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { PrivateKey, constants } from '../';
```

Comment:
This eslint disabling is not needed, cause this import will be resolved, when build is completed.